spec/README.md

```diff
@@ -79,39 +79,39 @@ A reference to a _term_ looks like this.
 > The provisions of the stability policy are not in effect until
 > the conclusion of the technical preview and adoption of this specification.
 
-Updates to this specification will not change
-the syntactical meaning, the runtime output, or other behaviour
-of valid messages written for earlier versions of this specification
-that only use functions defined in this specification.
+Updates to this specification will not make any well-formed or valid _message_ invalid.
+
 Updates to this specification will not remove any syntax provided in this version.
-Future versions MAY add additional structure or meaning to existing syntax.
+
+Updates to this specification will not change the syntactical meaning
```

Comment:
```suggestion
Updates to this specification will not change the meaning
```

(Not sure how "syntactical meaning" is different from just "meaning" here)

Reply:
Syntactical meaning is: `.input` is an input declaration. 

"Meaning" can be read as "how a specific `.input` is understood", so `.input {$foo @bar=baz}` might "change meaning" if we allowed expression attributes and assigned "meaning" to the `@bar` one. That's why I used the circuitous and painful wording.

Perhaps:

```suggestion
Updates to this specification will not alter the role assigned to 
defined elements of the syntax, except to assign an interpretation to syntax marked as
"reserved for future standardization".
```